contracts/acl/ACL.sol

```diff
@@ -288,11 +287,19 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         _setPermissionManager(_manager, _app, _role);
     }
 
+    /**
+    * @dev Internal function to grants a permission with parameters
+    */
+    function _grantPermissionP(address _entity, address _app, bytes32 _role, uint256[] _params) internal onlyPermissionManager(_app, _role) {
```

Comment:
I feel more comfortable moving the modifier to the external functions, since that's where it lives for the other functions.

I don't think we'll be running into bytecode size limits with this app :).